src/app/public/modules/modal/modal.component.spec.ts

```diff
@@ -555,4 +555,19 @@ describe('Modal component', () => {
 
     closeModal(modalInstance);
   }));
+
+  it('should prevent click events from bubbling beyond host element', fakeAsync(function () {
```

Comment:
@Blackbaud-SteveBrush should we have a sanity test here that ensures that we do still register clicks on modals themselves and that the only time we don't is if you watch something behind the host?

Reply:
Good call. Done.